omniscidb/QueryEngine/CostModel/DataSources/DwarfBench.cpp

```diff
@@ -72,12 +72,12 @@ std::vector<Detail::Measurement> DwarfBenchDataSource::PrivateImpl::convertMeasu
 
 DwarfBenchDataSource::DwarfBenchDataSource()
     : DataSource(DataSourceConfig{
-          .dataSourceName = "DwarfBench",
-          .supportedDevices = {ExecutorDeviceType::CPU, ExecutorDeviceType::GPU},
-          .supportedTemplates = {AnalyticalTemplate::GroupBy,
-                                 AnalyticalTemplate::Join,
-                                 AnalyticalTemplate::Scan,
-                                 AnalyticalTemplate::Sort}})
+          .data_source_name = "DwarfBench",
+          .supported_devices = {ExecutorDeviceType::CPU, ExecutorDeviceType::GPU},
+          .supported_templates = {AnalyticalTemplate::GroupBy,
```

Comment:
There's an issue with this syntax on windows, unfortunately. Could you please use the old-school plain init lists?

Reply:
oh, hdk exists on windows.. =) i missed it